server/cluster/coordinator.go

```diff
@@ -310,6 +311,9 @@ func (c *coordinator) runUntilStop() {
 }
 
 func (c *coordinator) run() {
+	failpoint.Inject("runSchedulerCheckInterval", func() {
```

Comment:
Could we change the `Ticker` instead of `runSchedulerCheckInterval`? So we can keep `runSchedulerCheckInterval` as a constant.
For example , re define the ticker after L317
```
	failpoint.Inject("runSchedulerCheckInterval", func() {
		ticker = time.NewTicker(100 * time.Millisecond)
	})
```